client/quantum_serverless/core/__init__.py

```diff
@@ -75,6 +74,7 @@
 )
 from .pattern import (
     QiskitPattern,
+    Program,
```

Comment:
Thank you for fixing this, I think it is causing weird behavior in sphinx builds since we build from latest release.

Reply:
This weird behavior was due to we were building docs from release